.github/deploy/steps/99-Databricks-Secrets.ps1

```diff
@@ -0,0 +1,21 @@
+
+
+New-DatabricksScope -name $db_secrets_scope
+
+$new_keys = $()
+foreach ($key in $keystore) {
+  $new_keys += $key.name
+  databricks secrets put --scope $db_secrets_scope --key $key.name --string-value $key.key
+  Write-Host "  Added key '$($key.name)'"
+}
+
+$existing_keys = Convert-Safe-FromJson -text (databricks secrets list --scope $db_secrets_scope --output JSON)
+Throw-WhenError -output $existing_keys
+
+foreach($secret in $existing_keys.secrets){
+  if(-not $new_keys.Contains($secret.key)){
```

Comment:
why have this new variable "new_keys"? New_keys is just $keystore.name ? 
This could remove line 7.